eventDataScript.js

```diff
@@ -0,0 +1,296 @@
+            for (var i=0;i<eventNames.length;i++)
+            {
+                var val = eventNames[i].trim();
+                if (newEvent==val.toString())
+                {
+                    isAlreadyStored = true;
+                    break;
+                }
+            }
+        }
+    }
+
+    if (isAlreadyStored) //if season is already stored
+    {
+        $("#newEventConfirmation").html("<i>Event \'" + newEvent +"\' already exists.</i>");    
+    } else
+    {
+        //add to radioList
+        $("#newEventConfirmation").html("<i>Event " + "\'"+ newEvent +"\'"+" has been added.</i>"); //confirmation text
+        $("#eventRadioList").append("<label><input type=\"radio\" name=\"eventListItem\" value=\"" + newEvent + "\">" + newEvent +"</label><br>"); //add to radiolist
+
+        //save newSeason in cookie
+        var cvalue = getCookie(cookieName) + newEvent + 'Δ';
+        setCookie(cookieName,cvalue,750);
+
+        $("#newEventInputBox").val("");
+    }
+}
+
+//mirrors the deleteCheckedSeason function
+function deleteCheckedEvent()
+{
+    var currCheckedSeason = getCookie('currCheckedSeason');
+    //cookie --> 'deep space events=austinΔfit champsΔ...'
+    var cookieName = currCheckedSeason + ' events';
+    var cookieVal = getCookie(cookieName);
+    var currCheckedEvent = getCurrEvent();
+    var cnameEnding = currCheckedSeason + '/' + currCheckedEvent;
+    
+    //splices the eventList cookie and gets rid of the function to be deleted
+    var eventList = cookieVal.trim();
+    var startIndex = eventList.indexOf(currCheckedEvent);
+    var endIndex = startIndex + currCheckedEvent.length;
+    var newText = eventList.substring(0,startIndex) + eventList.substring(endIndex+1);
+    
+    setCookie(cookieName,newText,750); //updates event data cookie
+    setCookie('currCheckedEvent','',750); //last checked event was deleted, restart cookie
+    
+    //delete team names for that event
+    var cname = '';
+    var cookieList = document.cookie.split(';');
+    for (var i = 0; i < cookieList.length;i++)
+    {
+        var currCname = cookieList[i].trim().split('=')[0];
+        if (currCname.endsWith(cnameEnding))
+        {
+            cname = currCname;
+        }
+    }
+    deleteCookie(cname);
+
+    document.location.reload();
+}
+
+
+function setTeamTable(season, event)
+{
+    var currEvent = getCurrEvent();
+    var currSeason = getCookie('currCheckedSeason');
+    if (currEvent !== ' ')
+    {
+        $('#teamTableTitle').text('Teams for event "' + currEvent + '."');
+    }
+    $("#eventSpecificTeamTable").html('<tr><th><b>Team Number</b></th> <th><b>Team Name</b></th> </tr>');
+
+    var teamsJSON = JSON.parse(getCookie(event_data_cookie_name(season,currEvent)));
+    if (teamsJSON == '{}') return;
+    const pairs = Object.entries(teamsJSON);
+    for (var i = 0; i < pairs.length; i++)
+    {
+        var teamNumber = pairs[i][0];
+        var teamName = pairs[i][1];
```

Comment:
this does work, but I wonder if this would be simpler or easier to read:

```js
var keys = Object.keys(teamsJSON); // see above on variable name
for(var i = 0; i < keys; i++) { 
  var teamNumber = keys[i];
  var teamName = teamsJSON[teamNumber];
}
```